tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/util/BRunUtil.java

```diff
@@ -432,7 +432,7 @@ protected static void invokePackageInit(CompileResult compileResult, String pack
                 }
             };
 
-            Scheduler scheduler = new Scheduler(4, false);
+            Scheduler scheduler = new Scheduler( false);
```

Comment:
Formatting issue here "new Scheduler( false);" this has an extra space character